node/taskcommand.ts

```diff
@@ -31,7 +31,7 @@ export class TaskCommand {
             for (var key in this.properties) {
                 if (this.properties.hasOwnProperty(key)) {
                     var val = this.properties[key];
-                    if (val) {
+                    if (val != null) {
```

Comment:
They did something wrong, it should be skipped.  Writing code to enable passing the wrong data type encourages doing the wrong thing.  The compiler will encourage the right thing.  If they bypass that and do the wrong thing, then it not getting set will further encourage the right thing.  They will see it's not getting set, look at the docs/types and fix it.